src/tests/cmocka/test_pam_srv.c

```diff
@@ -387,6 +393,8 @@ static int pam_test_teardown(void **state)
                             pam_test_ctx->wrong_user_fqdn, 0);
     assert_int_equal(ret, EOK);
 
+    test_dom_suite_cleanup(TESTS_PATH, TEST_CONF_DB, TEST_DOM_NAME);
```

Comment:
Nowadays, I see three calls to `test_dom_suite_cleanup()` and I guess the last one from this file is a left over, but I might be wrong.

Reply:
Hi,

thanks for the hint. As you will see in the two new patches the main issue here imo was how to handle to `--no-cleanup` option since each test will now do the cleanup on its own. So I added support to run a single test.

bye,
Sumit

Reply:
I don't see the point in executing just a single test but I suppose it will have its benefits.

Reply:
With the `--no-cleanup` option you want to keep the directory with the cache and confdb file around for further inspection. This might be useful when developing new tests or when using tests to better understand how some functions are working.

I would argue that even before I added `test_dom_suite_cleanup()` to the teardown function here the option only hab limited use because all tests where using the same cache and confdb file, potentially overwriting entries from a previous test. So even if the files were kept it would be hard to relate a specific entry to a specific test.

Another use-case for running a single test is getting the debug output only for this test which hopefully make developing new tests and debugging failed tests more easy.

Reply:
Understood. Thanks for the explanation.